### ISSUE TITLE
Went from 300H/s to 220H/s using xmrMiner_0.3.0 vs xmrMiner_0.2.1

### ISSUE THREAD

**Issue by joe7dust** [fireice-uk/xmr-stak#1358]
Had to upgrade because of fork, and old miner getting rejected. Now the performance is shit, what can I do?

**Comment by psychocrypt** [comment]
Which params do you used with xmrMiner and please post your nvidia.txt you
used with xmr-stak

**Comment by joe7dust** [comment]
set launchParam=
set bFactor=8
set bSleep=5

I use --noNVIDIA because only the cpu part of that program ever worked for me. Then after xmrstak is running I get up to 150H/s off the CPU. This has not changed for me since the fork.

However I also used xmrMiner_0.2.1.exe with 300H/s on my 1050ti at the same time and now it is just 220H/s after I had to upgrade to xmrMiner_0.3.0. The settings above are in xmrMiner_easyStart.bat
